command-expansion-server/src/packages/lib/CommandTypeCodegen.ts

```diff
@@ -174,7 +174,17 @@ ${doc}
 function mapArgumentType(argument: ampcs.FswCommandArgument, enumMap: ampcs.EnumMap): string {
   switch (argument.arg_type) {
     case "enum":
-      return enumMap[argument.enum_name].values.map((value) => `'${value.symbol}'`).join(" | ");
+      // boolean enum shouldn't be "TRUE | FALSE" but of `boolean` type
+      if (
+        enumMap[argument.enum_name].values.length === 2 &&
+        enumMap[argument.enum_name].values.filter(({ symbol, numeric }) => {
+          return symbol.toLocaleLowerCase() !== "true" && symbol.toLocaleLowerCase() !== "false";
+        }).length === 0
```

Comment:
The double negative here (with the `!==` and the "array is empty) confuses me at first glance but it makes sense.